schemas/threat.yml

```diff
@@ -0,0 +1,131 @@
+---
+- name: threat
+  title: Threat
+  group: 2
+  short: Fields describing detected threats and threat intel data
+  description: >
+    ECS threat fields encompass detected suspicious activity (e.g. IDS alert, malware activity) typically
+    reported by IDS/IPS and NGFWs, as well as threat intel/global IOC data ingested from a feed source like MISP or Anomali.
+
+  type: group
+  fields:
+
+    - name: id
+      level: extended
+      type: keyword
+      short: Threat ID
+      description: >
+        Threat ID, typically provided by reporting system (e.g. NGFW or IDS) or threat intel provider (Anomali, OTX, etc.)
+
+      example: 33227 
+
+    - name: name
+      level: extended
+      type: keyword
+      multi_field:
+        type: text
+        name: text
+      short: Name or classification of the threat detected or reported
+      description: >
+        Threat name, typically provided by reporting system (e.g. NGFW or IDS) or threat intel provider (Anomali, OTX, etc.)
+
+      example: ntp reserved mode denial of service
+
+    - name: tactic.name
+      level: extended
+      type: keyword
+      multi_field:
+        type: text
+        name: text
+      short: Threat tactic 
+      description: >
+          Name of the type of tactic used by this threat. You can use the Mitre ATT&CK Matrix Tactic categorization, for example.
```

Comment:
I think we need to step back a little more from ATT&CK in our descriptions, and clearly state that "tactic" is the broader category, and "technique" is a subcategory of how a tactic has been carried. 

This will be helpful for people using a different categorization framework.

Reply:
reorganized to move tactic &  technique top the bottom (grouping reported fields at the top vs potentially evaluated fields to the bottom)